src/app/level-navigate/level-navigate.component.scss

```diff
@@ -0,0 +1,18 @@
+@import "../styles/main";
+
+#level-navigate {
+  display: inline-block;
+  position: absolute;
+  top: 50%;
+  right: 0;
```

Comment:
I think we have a variable for 0.. $zero I guess.

Reply:
Currently, we did not have this variable. So added it.